src/report-links/report-links.test.tsx

```diff
@@ -56,192 +88,54 @@ describe("Report Links", () => {
   });
 
   it("should show all report links", () => {
-    const mockReports = [
-      {
-        name: "First Test Report",
-        uuid: "report-uuid"
-      }
-    ];
     const { queryByText } = render(
       <ReportLinks
         {...commonWidgetProps}
         title="My Test Report Links"
-        reports={mockReports}
+        charts={mockCharts}
       />
     );
 
     expect(queryByText("First Test Report")).toBeTruthy();
   });
 
-  it("should request for new report when play button clicked", async () => {
-    const mockReports = [
-      {
-        name: "First Test Report",
-        uuid: "report-uuid"
-      }
-    ];
-
-    mockEsmAPI.openmrsFetch.mockResolvedValueOnce({
-      data: { uuid: "report-request-uuid" }
-    });
-
-    const expectedRequestOptions = {
-      method: "POST",
-      body: {
-        status: "REQUESTED",
-        priority: "HIGHEST",
-        reportDefinition: {
-          parameterizable: { uuid: "report-uuid" }
-        },
-        renderingMode:
-          "org.openmrs.module.reporting.web.renderers.DefaultWebRenderer"
-      },
-      headers: { "Content-Type": "application/json" }
-    };
-
-    const { container } = render(
-      <ReportLinks
-        {...commonWidgetProps}
-        title="My Test Report Links"
-        reports={mockReports}
-      />
-    );
-
-    const firstReportLink = getByTitle(container, "Request Report");
-    await act(async () => {
-      firstReportLink.click();
-    });
-
-    const expectedReportRequestUrl = "/ws/rest/v1/reportingrest/reportRequest";
-
-    expect(mockEsmAPI.openmrsFetch).toBeCalledWith(
-      expectedReportRequestUrl,
-      expectedRequestOptions
-    );
-  });
-
-  it("should show loading spinner when report request is processing", async () => {
-    const mockReports = [
-      {
-        name: "First Test Report",
-        uuid: "report-uuid"
-      }
-    ];
-    mockEsmAPI.openmrsFetch.mockResolvedValueOnce({
-      data: { uuid: "report-request-uuid" }
-    });
-
+  it("should not render modal when report link is not clicked", () => {
```

Comment:
can it be named like 
```suggestion
  it("should not show chart modal during initial load", () => {
```